src/modules/suggestions/utilities/delimiters.ts

```diff
@@ -27,24 +27,10 @@ const delimiters: Delimiters = {
 function getLastDelimiterInUrl(url: string): Delimiter {
   const symbols = Object.values(delimiters);
   symbols.forEach(key => {
-    const prevCharIndex = url.lastIndexOf(key.symbol) - 1;
-    key.index = isSecondLastCharADelimiter(url.charAt(prevCharIndex)) ? url.lastIndexOf(key.symbol)-1 :
-      url.lastIndexOf(key.symbol);
+    key.index = url.lastIndexOf(key.symbol);
```

Comment:
This whole method feels overly complex for what seems like a simple operation best done by iterating from the end of the url string, unless there's something I'm missing here.

```
  function getLastDelimiterInUrl(url: string): Delimiter {
    const symbols: Delimiter[] = Object.values(delimiters);
    for (let i = url.length -1 ; i >= 0; i--) {
      const symbol = symbols.find((s) => s.symbol === url[i]);
      if (symbol) {
        symbol.index = i;
        return symbol;
      }
    }
  }
}
```

This version will stop as soon as it finds a match, rather than requiring O2n complexity